randovania/games/samus_returns/assets/hash_words.json

```diff
@@ -0,0 +1,120 @@
+[
+  "Ability",
+  "Aeion",
+  "Alpha",
+  "Amiibo",
+  "Arachnus",
+  "Armor",
+  "Artifact",
+  "Attack",
+  "Autoad",
+  "Autrack",
+  "Baby",
+  "Ball",
+  "Beam",
+  "Big",
+  "Blob",
+  "Bomb",
+  "Boost",
+  "Boots",
+  "Brain",
+  "Burst",
+  "Charge",
+  "Checkpoint",
+  "Chozo",
+  "Chute",
+  "Closed",
+  "Crumble",
+  "DNA",
+  "Diggernaut",
+  "Door",
+  "Drift",
+  "Drivel",
+  "Energy",
+  "Federation",
+  "Fly",
+  "Fusion",
+  "Galactic",
+  "Gamma",
+  "Gawron",
+  "Gigadora",
+  "Glow",
+  "Grapple",
+  "Gravitt",
+  "Gravity",
+  "Green",
+  "Gryncore",
+  "Gullug",
+  "Gunship",
+  "Gunzoo",
+  "Halzyn",
+  "Hard",
+  "Hatchling",
+  "Hazardous",
+  "Heat",
+  "High",
+  "Hornoad",
+  "Husk",
+  "Ice",
+  "Icicle",
+  "Inactive",
+  "Item",
+  "Jump",
+  "Larva",
+  "Leech",
+  "Lightning",
+  "Liquid",
+  "Map",
+  "Meboid",
+  "Metroid",
+  "Missile",
+  "Moheek",
+  "Morph",
+  "Mother",
+  "Moto",
+  "Mumbo",
+  "Normal",
+  "Omega",
+  "Orb",
+  "Paraby",
+  "Phase",
+  "Pirates",
+  "Plants",
+  "Plasma",
+  "Power",
+  "Proteus",
+  "Pulse",
+  "Queen",
+  "Ramulken",
+  "Recharge",
+  "Red",
+  "Reserve",
+  "Ridley",
+  "Rock",
+  "SR388",
+  "Samus",
+  "Save",
+  "Scan",
+  "Screw",
+  "Seal",
+  "Space",
+  "Spazer",
+  "Sphere",
+  "Spider",
+  "Spring",
+  "Station",
+  "Suit",
+  "Super",
+  "Tank",
+  "Taramarga",
+  "Teleport",
+  "Thrower",
+  "Tsumuri",
+  "Varia",
+  "Wallfire",
+  "Water",
+  "Wave",
+  "Yellow",
+  "Zebes",
+  "Zeta"
```

Comment:
can you add a trailing comma please, just in case that more words will get added in the future?

Reply:
invalid json

Reply:
ah this is json, not python code, whoops.
Disregard